integreat_cms/cms/forms/regions/region_form.py

```diff
@@ -185,6 +185,20 @@ def __init__(self, *args, **kwargs):
         self.fields["deepl_midyear_start_enabled"].initial = bool(
             self.instance.deepl_midyear_start_month
         )
+        non_disableable_offers = (
+            (
+                offer_template
+                for offer_template in Page.objects.filter(region=self.instance)
+                .values_list("embedded_offers", flat=True)
+                .distinct()
+                if offer_template
+            )
+            if self.instance
+            else []
+        )
+        self.fields["offers"].queryset = self.fields["offers"].queryset.exclude(
+            id__in=non_disableable_offers
+        )
```

Comment:
This will do the opposite of what we want: If an offer is removed from the queryset, it is automatically disabled when the form is submitted.